gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -150,7 +150,7 @@ def bread(start_sector, number_sectors)
       encoded_data = @parser.output_to_attribute(run_correct_powershell(read_script))
       t2           = Time.now.getlocal
       buffer       = ""
-      Base64.decode64(encoded_data).split(' ').each { |c| buffer += c.to_i.chr }
+      Base64.decode64(encoded_data).split(' ').each { |c| buffer << c.to_i.chr }
```

Comment:
@roliveri I'm really sorry that I did not document my original investigation into this for posterity, but I did spend quite some time of the original SCVMM SmartState Feature investigation on this very subject.  I am going through some other Powershell scenarios in an attempt to come up with an alternate solution. Rest assured that I am *not* inserting spaces into the data - that is the being done by Powershell.   My assumption is that is because of the necessity to read the data into an array of bytes constructed via New-Object System.Byte[] with the buffer size.  When you print that array out in Powershell, a newline (carriage return?) is placed after every byte.

Here is another example - 

$buf contains the file data.
$filecontentbytes = [System.Text.Encoding]::UTF8.GetBytes($buf)
$filecontentencoded = [System.Convert]::ToBase64String($filecontentbytes)

$filecontentencoded is sent to STDOUT of the winrm protocol.
running Base64.decode64 on that data results in the same thing as with the implemented solution - for example - 

"118 104 100 120 102 105 108 101 77 0 105 0 99 0 114 0 111 0 115 0 111 0 102 0 116 0 32 0 87 0 105 0 110 0 100 0 111 0 119 0 115 0 32 0 54 0 46 0 51 0 46 0 57 0 54 0 48 0 48 0 46 0 49 0 54 0 53 0 50 0 49 0 0 0 0 0 0 0 0 0 0 0 ....."
etc.